apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -260,7 +234,7 @@ var TopInstructions = React.createClass({
                 headerHasFocus={false}
                 onClick={this.handleDocumentationClick}
               />}
-            {(experiments.isEnabled('resources_tab') || experiments.isEnabled('resourcesTab')) &&
```

Comment:
what is the history of `resources_tab` -- is it just a synonym for `resourcesTab`?

Reply:
resources_tab was what I used to do the A/B experiment. I wanted to make sure that people and tests could still use the ?enableExperiments=resourcesTab method to get to the resources tab and I wasn't sure that would be possible if I used resourcesTab as the keyphrase for the A/B part of the experiment.